i18n/pt.json

```diff
@@ -125,14 +126,14 @@
   "I18N_LIBRARY_CATEGORIES_WELCOME": "Bem-vindo",
   "I18N_LIBRARY_CREATE_EXPLORATION_QUESTION": "Gostarias de criar uma?",
   "I18N_LIBRARY_GROUPS_COMPUTING": "Informática",
-  "I18N_LIBRARY_GROUPS_FEATURED_ACTIVITIES": "Actividades em Destaque",
+  "I18N_LIBRARY_GROUPS_FEATURED_EXPLORATIONS": "Explorações em destaque",
```

Comment:
This seems to have been a reversion? In develop I think it is FEATURED_ACTIVITIES.

If your branch is messed up, might be worth redoing from scratch...